apps/src/lib/ui/SystemDialog/Field.jsx

```diff
@@ -0,0 +1,79 @@
+import React, {PropTypes} from 'react';
+import color from '@cdo/apps/util/color';
+import {createUuid} from "../../../utils";
+
+/**
+ * Field wrapper for a "System" dialog style.
+ *
+ * Can be given any (or none) of a label, labelDetails, and/or error - all
+ * plain text.  They'll be formatted properly and wrap the field as needed.
+ * Takes a single child, which should be a form control
+ * Will automatically set an id on the input element to bind the label to it
+ * properly.
+ */
+export default class Field extends React.Component {
+  static propTypes = {
+    label: PropTypes.string,
+    labelDetails: PropTypes.string,
+    error: PropTypes.any,
+    children: PropTypes.element.isRequired,
+    style: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+    this.uniqueId = createUuid();
+  }
+
+  render() {
+    const {label, labelDetails, error, children, style} = this.props;
+    return (
+      <div
+        style={{
+          marginBottom: 15,
+          ...style,
+        }}
+      >
+        {(label || labelDetails) &&
+        <label
+          htmlFor={this.uniqueId}
+          style={{
+            display: 'block',
+            color: color.charcoal,
+          }}
+        >
+          {label &&
+          <span style={{fontWeight: 'bold', marginRight: '1em'}}>
```

Comment:
nit: indentation here. i don't see it in the styleguide, but i will tend to format JSX with conditionals like:

```
{label &&
  <Component>
    <Child/>
  </Component>
}
```

rather than:

```
{label &&
<Component>
  <Child/>
</Component>
}
```